src/components/description_list/description_list_title.styles.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { css } from '@emotion/react';
+import {
+  euiFontSize,
+  euiTextBreakWord,
+  logicalTextAlignCSS,
+  euiBreakpoint,
+  logicalCSS,
+} from '../../global_styling';
+import { tint, UseEuiTheme } from '../../services';
+import { euiTitle } from '../title/title.styles';
+
+export const euiDescriptionListTitleStyles = (euiThemeContext: UseEuiTheme) => {
+  const { euiTheme, colorMode } = euiThemeContext;
+
+  const columnDisplay = `
+    ${logicalCSS('width', '50%')} // Flex-basis doesn't work in IE with padding
+    ${logicalCSS('padding-right', euiTheme.size.s)}
+  `;
+  return {
+    euiDescriptionList__title: css`
+      ${euiTextBreakWord()};
```

Comment:
Minor nit, this util has its own semicolons so no need to add another after
```suggestion
      ${euiTextBreakWord()}
```